lib/zendesk_api/resources.rb

```diff
@@ -857,4 +857,12 @@ class Variant < ZendeskAPI::Resource
       has_many Variant
     end
   end
+
+  class PushNotificationDevice < Resource
+    def self.destroy_many(client, tokens)
+      ZendeskAPI::Collection.new(client, self,"push_notification_devices" => tokens,
+                                 :path =>  "push_notification_devices/destroy_many",
+                                 :verb => :post).fetch
```

Comment:
Yes, don't call fetch. Collections are lazy-loaded.

Please also indent everything only 2 spaces off the top line.